### PR TITLE
Add a description to the focus dialog

### PR DESCRIPTION
**Pull request by mcclure**
I am still a little shaky on the XML layouts so I did it programmatically. Dunno if that's OK or not.

### CODE REVIEW

**Comment by mcclure** [comment]
Screenshot_20220930-233954

Screenshot_20220930-234029

**Review by connyduck — COMMENTED** [review]
Having half the layout in code and the other half in xml is very weird. Can you please change it so we have it in one place (I'd prefer xml)?

**Comment by mcclure** [comment]
OK, otherwise no objection to the look/text?

**Comment by connyduck** [comment]
No, I like it!

**Comment by mcclure** [comment]
Redid the PR to use XML only. Seems to look exactly the same with less code required.

**Comment by mcclure** [comment]
Screenshot_20221012-221157

Screenshot_20221012-223750

**Comment by mcclure** [comment]
Build failed but it appears to be a CI configuration issue, not the PR?

> Could not resolve all files for configuration ':app:greenDebugCompileClasspath'.
   > Could not resolve com.github.UnifiedPush:android-connector:2.0.1.

File: app/src/main/res/layout/dialog_focus.xml [hunk 1]
```diff
@@ -1,17 +1,35 @@
...
+        android:layout_height="wrap_content"
+        android:layout_gravity="center_horizontal"
+        android:padding="24dp"
+        android:gravity="center"
+        android:textSize="18sp"
+        android:text="@string/set_focus_description" />
 
-    <com.keylesspalace.tusky.components.compose.view.FocusIndicatorView
-        android:id="@+id/focusIndicator"
+    <FrameLayout xmlns:android="http://schemas.android.com/apk/res/android"
```
Comment (connyduck) [inline app/src/main/res/layout/dialog_focus.xml:17]:
```suggestion
    <FrameLayout
```

**Comment by connyduck** [comment]
> Build failed but it appears to be a CI configuration issue, not the PR?

Yes, retrying fixed it

**Review by connyduck — APPROVED** [review]

**Pull request merged**